### DISCUSSION TITLE
any way to enable mouse click and drag + right click for web apps?

### DISCUSSION THREAD

**Discussion by DukeStamina** [rocksdanister/lively#824] [Q&A]
I'm really wanting to use https://milanote.com/ as a web-app url-based wallpaper (is that redundant? lmao) on my second monitor. To  fully utilize the features of the website correctly I need to be able to use keyboard + L mouse click & drag + R Mouse click

lemme know if there's any way of being able to do this within lively <3

if not, any recommendations on how I might be able to do this would be MUY APPRECIADO!

**Comment by rocksdanister** [comment]
![Screenshot 2021-10-20 032809](https://user-images.githubusercontent.com/17554161/137996503-69789523-83e2-4ffa-9b18-8aa144e59a95.png)

Keyboard + Mouse can be enabled in settings (only Mouse is enabled by default.).. desktop icons will be hidden in this mode to avoid selecting them by mistake.

Right Mouse click is not enabled implemented because rightclick menu shows Windows contextmenu:
![Screenshot 2021-10-20 032939](https://user-images.githubusercontent.com/17554161/137996668-05dc6bde-83e9-4cf7-a623-1e22d7258a17.png)

**Comment by DukeStamina** [comment]
if this hasn't been implemented due to not wanting to override the windows desktop rightclick + click and drag....
then maybe I could suggest a cool feature would be to be able to choose within the app which of the 2 (web app or windows) to prioritize, and then allow a key or key combo to be used in order to override the prioritized choice.

like holding alt lets you use the windows context menu if the web app is prioritized

**Reply by rocksdanister** [reply]
In theory there is actually an easier way.. just make the wallpaper same layer as the desktop icons instead of behind that is being drawn now.
Desktop icons will disappear though because wallpaper will be rendered ontop of it.

**Reply by DukeStamina** [reply]
is your suggestion something that I would be able to do within the software, or is that more of a developer thing?

**Reply by rocksdanister** [reply]
developer.

**Comment by DukeStamina** [comment]
oh damn such a fast response! so I'm using it now and I can totally click and use keyboard, but click and drag doesn't seem to work. is that normal?

I AM on win 11 (like a damn fool) if that info helps at all

**Reply by DukeStamina** [reply]
> _the website also only seems to work when the engine is set to MS Edge, CEF only displays a blank white screen_

edited:
**SCRATCH THAT, I JUST NEEDED TO ALLOW CEF THRU MY FIREWALL**
click and drag is still not usable tho
